### PR TITLE
Recommitting "Updating 'Club Penguin Rewritten' to 3.0.0" to Resolve Errors

### PR DESCRIPTION
**Pull request by BartBoys3**
Solved an additional issue where talking to Sensei would report as being in a party room as talking to him is treated as a room within the game. The room IDs 951 (Standard Sensei) and 953 (Fire Sensei) have been added to the triggers for Dojo and Fire Dojo respectively. 
More screenshots:
image
image
image
image
image
*There's nothing to show for the new change as the it looks exactly the same as if you were in the Dojo/Fire Dojo not talking to Sensei/Fire Sensei.

### CODE REVIEW

File: Club Penguin Rewritten/dist/metadata.json [hunk 1]
```diff
@@ -8,17 +8,17 @@
     "en": "A new free virtual world based on Disney's Club Penguin with weeky updates. Waddle around and meet new friends!"
   },
   "url": "play.cprewritten.net",
-  "version": "3.0.0",
+  "version": "3.0.2",
```
Comment (Alanexei) [inline Club Penguin Rewritten/dist/metadata.json:11]: Change the version to `3.0.1`

Comment (BartBoys3) [reply Club Penguin Rewritten/dist/metadata.json:11]: I was told by forzaum to do 3.0.2 because the revert was 3.0.1

Comment (BartBoys3) [reply Club Penguin Rewritten/dist/metadata.json:11]: #1461

Comment (doomlerd) [reply Club Penguin Rewritten/dist/metadata.json:11]: Resolve the conflicting files.

Comment (BartBoys3) [reply Club Penguin Rewritten/dist/metadata.json:11]: what conflicting files? what exactly would resolving that look like?

Comment (Cairo2k18) [reply Club Penguin Rewritten/dist/metadata.json:11]: @BartBoys3 
image

Comment (BartBoys3) [reply Club Penguin Rewritten/dist/metadata.json:11]: I see the button, but when I press it, I don't see what to change.

Comment (BartBoys3) [reply Club Penguin Rewritten/dist/metadata.json]: Conflicts have been resolved

**Pull request merged**